genrl/deep/common/trainer.py

```diff
@@ -513,7 +513,7 @@ def train(self) -> None:
             if epoch % self.log_interval == 0:
                 self.logger.write(
                     {
-                        "Timestep": epoch * self.agent.rollout_size,
+                        "timestep": epoch * self.agent.rollout_size,
```

Comment:
We have used `"timestep"` in rest of the trainer.py file and also in the logger.py file.
Should I change it to `"Timestep"` in all the places to maintain uniformity with `"Episode"`

Reply:
No, better to have everything be in lower case imo

Reply:
Should I change "`Episode`"to `"episode"` then? Also, are any more changes required?

Reply:
Yeah. Will review the rest a little later.